assets/js/apps/search/components/Result.jsx

```diff
@@ -58,18 +58,21 @@ function Result({ result }) {
               type="button"
               onClick={dataToggler}
               className="btn btn-link"
-              data-target="#data"
+              data-bs-target="#data"
               name="showChild"
             >
               <i className={dataToggleClass ? 'fa fa-plus' : 'fa fa-minus'} />
             </button>
             <a style={{ fontWeight: 'bold' }} href={result.url}>{result.title}</a>
             <ul
-              className="nav nav-pills badgelist pull-right"
+              className="nav nav-pills badgelist float-end"
               style={{ display: 'inline-block' }}
             >
-              <li className="pull-right">
-                <span className="badge badge-info mr-2" title={`Date Created: ${result.created_time}`}>
+              <li className="float-end">
+                <span
+                  className="badge bg-info me-2"
+                  title={`Date Created: ${result.created_time}`}
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 30 spaces but found 18.](https://app.codacy.com/gh/mytardis/mytardis/pullRequest?prid=7933941)
```suggestion
                              title={`Date Created: ${result.created_time}`}
```